test/module.test.js

```diff
@@ -1,106 +1,229 @@
     expect(html).toMatchSnapshot()
   })
+
   test('simple atom', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/simple_atom'))
+    nuxt = await setupNuxt({
+      ...config,
+      ...{
+        feed: [
+          createFeed('atom1')
+        ]
+      }
+    })
+
     const html = await get('/feed.xml')
     expect(html).toMatchSnapshot()
   })
+
   test('simple json', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/simple_json'))
+    nuxt = await setupNuxt({
+      ...config,
+      ...{
+        feed: [
+          createFeed('json1')
+        ]
+      }
+    })
+
     const html = await get('/feed.xml')
     expect(html).toMatchSnapshot()
   })
 
   test('non-latin rss', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/non_latin_rss'))
+    nuxt = await setupNuxt({
+      ...config,
+      ...{
+        feed: [
+          {
+            create(feed) {
+              feed.options = {
+                title: 'Популярные новости России и мира',
+                link: 'http://site.ru/feed.xml',
+                description: 'Новости России и мира на сайте site.ru',
+                updated: new Date(Date.UTC(2000, 6, 14))
+              }
+              feed.addContributor({
+                name: 'Команда проекта site.ru',
+                email: 'support@ site.ru',
+                link: 'http://site.ru/'
+              })
+            },
+            type: 'rss2'
+          }
+        ]
+      }
+    })
+
     const html = await get('/feed.xml')
     expect(html).toMatchSnapshot()
   })
 
-  test('no type set', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/no_type'))
-    expect(await get('/feed.xml')).toMatchSnapshot()
-  })
-
   test('object rss', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/object_rss'))
+    nuxt = await setupNuxt({
+      ...config,
+      ...{
+        feed: {
+          data: { title: 'Feed Title' },
+          create(feed, data) {
+            feedOptions.title = data.title
+            feed.options = feedOptions
+          },
+          type: 'rss2'
+        }
+      }
+    })
+
     const html = await get('/feed.xml')
     expect(html).toMatchSnapshot()
   })
 
   test('factory rss', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/factory_rss'))
+    nuxt = await setupNuxt({
+      ...config,
+      ...{
+        feed: {
+          data: { title: 'Feed Title' },
+          factory: data => ({
+            data,
+            create(feed, { title }) {
+              feedOptions.title = data.title
+              feed.options = feedOptions
+            },
+            type: 'rss2'
+          })
+        }
+      }
+    })
+
     const html = await get('/feed.xml')
     expect(html).toMatchSnapshot()
   })
 
   test('function rss', async () => {
-    nuxt = await setupNuxt(require('./fixture/configs/function_rss'))
+    nuxt = await setupNuxt({
+      ...config,
+      ...{
```

Comment:
unnecessary spreading (same for other tests)

```js
{
  ...config,
  feed: () => createFeed()
}
```

Reply:
@manniL In fact, it is necessary to cover the entire module

Reply:
I maintain a configuration file and modify it with each test to increase coverage

Reply:
I don't think you got me right:

### What you do

```js
 ...config,
      ...{
        feed: [
          { ...createFeed(), ...{ path: '/feed1.xml' } },
          { ...createFeed(), ...{ path: '/feed2.xml' } }
        ]
      }
```

### What I suggested

```js
 ...config,
    feed: [
      { ...createFeed(), ...{ path: '/feed1.xml' } },
      { ...createFeed(), ...{ path: '/feed2.xml' } }
    ]
```

Reply:
Sorry, I got it wrong, changed